src/commands/moderation/ban.ts

```diff
@@ -0,0 +1,56 @@
+import { botCache, Member, ban, sendMessage } from "./../../../deps.ts";
+import { Embed } from "./../../utils/Embed.ts";
+import { sendEmbed } from "./../../utils/helpers.ts";
+
+botCache.commands.set(`ban`, {
+  name: `ban`,
+  guildOnly: true,
+  arguments: [
+    {
+      name: "member",
+      type: "member",
+      missing: (message) => {
+        return sendMessage(message.channelID, "User not found!");
+      },
+    },
+    {
+      name: "days",
+      type: "number",
+      defaultValue: 0,
+    },
+    {
+      name: "reason",
+      type: "...string",
+      defaultValue: "No reason given",
+    },
+  ],
+  userServerPermissions: ["BAN_MEMBERS"],
+  botServerPermissions: ["BAN_MEMBERS"],
+  execute: async (message, args: BanArgs) => {
+    try {
+      await ban(message.guildID, args.member.id, {
```

Comment:
Before, banning we should have a few checks in place. 

1. A bot can't ban someone higher than them. Use the highestRole() to get the highest role of the bot and the args.member to compare. 
2. A user should never be able to ban someone higher than them. For example, a mod shouldnt be able to ban an admin. Same concept as #1 but this time compare member and message.author
3. A ban usually should be first accompanied by a dm of the reason to the user so before the await ban we should have a sendDirectMessage()